components/_expanders/expander.js

```diff
@@ -0,0 +1,39 @@
+const expander = {
```

Comment:
i need some clarification here, is this one expander? or is an expander configuration? a configuration should probably only be used to init the component for the documentation, and not be rendered in the main solution.

Reply:
I am not sure what you mean its for all expanding components.

This JS file needs to be shipped to our main solution so that we have the expanding functionality. This does not work for the documentation part of the design system yet.

The webpack bundling for JS to work within our documentation has just been merged.  
To add my JS to the webpack bundle I would need to create what you have coined "Configuration" or what Stephan called "Initialization" (We need to align on terminology/names) JS file and add a reference to that file in design-system.js